src/layers/legacy/medCoreLegacy/parameters/medTimeLineParameterL.cpp

```diff
@@ -66,11 +66,6 @@ class medTimeLineParameterLPrivate
 
         if(numberOfFramesLabel)
             delete numberOfFramesLabel;
-
```

Comment:
If you remove these lines and getWidget is never call, ok in normal way this never happens, you create a memory leak.
Because in this case extensionShiftParameter will never parented.
Parent the extensionShiftParameter on **this** during the construction phase is the only good way.